package.json

```diff
@@ -2,9 +2,10 @@
   "name": "eslint-config-seek",
   "version": "10.2.0",
   "description": "ESLint configuration used by SEEK",
-  "main": ".eslintrc.js",
+  "main": "index.js",
   "files": [
-    ".eslintrc.js"
```

Comment:
I would also remove `.eslintrc.js` from the repo, and keep only `index.js` and `base.js`.